R/likelihood.R

```diff
@@ -1,93 +1,104 @@
-#' Estimate the (log) likelihood for observed branching processes
+#' Estimate the log-likelihood/likelihood for observed branching processes
 #'
 #' @inheritParams simulate_summary
-#' @param chains Vector of sizes/lengths of transmission chains.
-#' @param nsim_obs Number of simulations if the likelihood is to be
-#' approximated for imperfect observations.
-#' @param log Logical; Should the results be log-transformed? (Defaults
-#' to TRUE).
+#' @inheritParams offspring_ll
+#' @param nsim_obs Number of simulations if the log-likelihood/likelihood is to
+#' be approximated for imperfect observations.
+#' @param log Logical; Should the log-likelihoods be transformed to
+#' likelihoods? (Defaults to TRUE).
 #' @param obs_prob Observation probability (assumed constant)
 #' @param exclude A vector of indices of the sizes/lengths to exclude from the
-#' likelihood calculation.
-#' @param individual If TRUE, a vector of individual (log)likelihood
+#' log-likelihood calculation.
+#' @param individual If TRUE, a vector of individual log-likelihood/likelihood
 #' contributions will be returned rather than the sum.
-#' @param ... Parameters for the offspring distribution.
```

Comment:
This is now seems to be inherited from `simulate_summary` - should it not, if anything, be inherited from `offspring_ll`? I'm not sure how to inherit selectively - perhaps change the order?